extensions/projection/schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema":"http://json-schema.org/draft-07/schema#",
+  "$id":"schema.json#",
+  "title":"Projection Extension",
+  "description":"STAC Projection Extension to a STAC Item",
+  "allOf":[
+    {
+      "$ref": "https://raw.githubusercontent.com/radiantearth/stac-spec/master/item-spec/json-schema/item.json#/definitions/core"
+    },
+    {
+      "$ref":"#/definitions/proj"
+    }
+  ],
+  "definitions":{
+    "proj":{
+      "properties":{
+        "properties":{
+          "type":"object",
+          "required":[
+            "proj:epsg"
+          ],
+          "properties":{
+            "proj:epsg":{
+              "title":"EPSG code",
+              "type":"integer"
+            },
+            "proj:crs":{
+              "title":"Coordinate Reference System in PROJ format",
+              "type":"string"
+            },
+            "proj:geometry":{
+              "title":"Geometry",
+              "type":"object"
+            },
+            "proj:bbox":{
+              "title":"Extent",
+              "type":"array",
+              "minItems":4,
+              "maxItems":6,
+              "items":{
+                "type":"number"
+              }
+            },
+            "proj:centroid":{
+              "title":"Centroid",
+              "type":"array",
+              "minItems":2,
+              "maxItems":3,
```

Comment:
Why is maxItems set to 3 and not 2? I don't see an explanation in the spec what the third element should be.